helm-chart/kuberay-operator/templates/role.yaml

```diff
@@ -75,33 +108,50 @@ rules:
   - patch
   - update
 - apiGroups:
-  - networking.k8s.io
```

Comment:
Looks like ingress are removed?

Reply:
I just copy from the https://github.com/ray-project/kuberay/blob/7ecdd22c5812db8754e470635b20d9ee4a6a425d/ray-operator/config/rbac/role.yaml let me double check

Reply:
This is outdated. Used master instead

Reply:
resolved.